region/brinstar/red/Beta Power Bomb Room.json

```
@@ -137,6 +137,55 @@
         "If any Sidehoppers remain after using a Power Bomb, safely kill them from the hole below."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Morph Ball Dodge)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
+        "Morph"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 17,
+          "openEnd": 0
```

From the spot you were showing me
```suggestion
          "openEnd": 1
```